src/GitHub.Api/Git/Repository.cs

```diff
@@ -134,6 +134,11 @@ public ITask ReleaseLock(string file, bool force)
             return repositoryManager.UnlockFile(file, force);
         }
 
+		public ITask CheckoutFiles( List<string> files )
+		{
+			return repositoryManager.CheckoutFiles( files );
+		}
+
```

Comment:
This should be indented with spaces. There should be no spaces after `(` and before `)`